d2exporter/export.go

```diff
@@ -131,6 +131,9 @@ func toShape(obj *d2graph.Object, theme *d2themes.Theme) d2target.Shape {
 	if obj.LabelPosition != nil {
 		shape.LabelPosition = *obj.LabelPosition
 	}
+	if obj.MaskLabel != nil {
```

Comment:
this shouldn't be in exporter. this is purely a render thing (e.g. if one day someone uses d2 diagrams rendered to Canvas or d3, this shouldn't appear).

i'm not sure how else to mark it though. Maybe you can create a `sequenceAttrs` (make sure to omitempty).

even currently, why are there a bunch of "false" in the testdata if u have omitempty?